Composer/packages/client/src/recoilModel/DispatcherWrapper.tsx

```diff
@@ -123,7 +123,7 @@ export const DispatcherWrapper = ({ children }) => {
         if (filePersistence.isErrorHandlerEmpty()) {
           filePersistence.registerErrorHandler(setProjectError);
         }
-        filePersistence.notify(assets, previousAssets);
+        await filePersistence.notify(assets, previousAssets);
```

Comment:
@srinaath is this change required?

Reply:
@hatpick Nope we shouldn't be requiring to await on filePersistence. This notify should be more like a fire and forget model where a task is added to the taskQueue and thigns are processed in the background.

@alanlong9278 Let's try to avoid awaiting here as we do not know the side effects of this change. In addition, why do you need to avoid?

Reply:
Remove it. Add `await` is for test updating file content sync. We should remove it.